tests/lib/bootstrap.js

```diff
@@ -102,6 +107,19 @@ function jqueryUrl() {
 	return url;
 }
 
+function migrateUrl() {
+	var version = parseUrl().migrate;
+	var url;
+
+	if ( version === "git" ) {
+		url = "http://code.jquery.com/jquery-migrate-" + version;
+	} else {
+		url = "../../../external/jquery-migrate-" + ( version || "1.12.4" ) + "/jquery-migrate";
```

Comment:
There is no `jquery-migrate-1.12.4`. But we also don't need a default version here since the migrate plugin isn't loaded by default and you have to explicitly specify the version to include.

This also needs to be registered with QUnit so that a UI element is generated. See `tests/lib/qunit.js` and search for `QUnit.config.urlConfig`. Since there's a 1:1 mapping between the jQuery version and the jQuery Migrate version, I'd prefer if the `migrate` config was a boolean. This function should look at the jQuery version to determine the jQuery Migrate version to load.

jQuery 1.x and 2.x -> jQuery Migrate 1.4.1
jQuery 3.x -> jQuery Migrate 3.0.0
jQuery Git -> jQuery Migrate Git